nntp/nntp.c

```diff
@@ -2654,6 +2655,7 @@ static bool nntp_msg_open(struct Mailbox *m, struct Message *msg, struct Email *
     }
 
     /* fetch message to cache file */
+    char buf[PATH_MAX] = { 0 };
```

Comment:
`PATH_MAX` is a bit misleading now because the buffer is not used for a path anymore. What do you think?

Reply:
true, but it still needs to be quite large for a message-id
```c
snprintf(buf, sizeof(buf), "ARTICLE %s\r\n", nntp_edata_get(e)->article_num ? article : e->env->message_id);
```
https://github.com/neomutt/neomutt/blob/07c62c30ad149c194440386bd8e43fb506413820/nntp/nntp.c#L2657C1-L2658

Reply:
The RFC doesn't seem to limit the message-id, so 2048 bytes?
(and leave converting the rest of the nntp driver for another day)

Reply:
👍🏼 I've used 2048 bytes now.